From c47efeda4c94af272638b4e035717f020447daa2 Mon Sep 17 00:00:00 2001
From: gongxiao-intel <xiaoyan.gong@intel.com>
Date: Tue, 22 Nov 2022 16:32:50 +0000
Subject: [PATCH 12/12] igc: fix other queue packets launch time being set.

---
 drivers/net/igc/igc_txrx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/igc/igc_txrx.c b/drivers/net/igc/igc_txrx.c
index 430928f9a2..811fd49ac4 100644
--- a/drivers/net/igc/igc_txrx.c
+++ b/drivers/net/igc/igc_txrx.c
@@ -1616,7 +1616,7 @@ igc_set_xmit_ctx(struct igc_tx_queue *txq,
 	ctx_txd->vlan_macip_lens = rte_cpu_to_le_32(vlan_macip_lens);
 	ctx_txd->mss_l4len_idx = rte_cpu_to_le_32(mss_l4len_idx);
 
-	if (ol_flags & RTE_MBUF_F_TX_IEEE1588_TMST) {
+	if (igc_timestamp_dynflag > 0 && txq->queue_id == 0) {
 		ctx_txd->u.launch_time = igc_tx_launchtime(txtime);
 	} else {
 		ctx_txd->u.launch_time = 0;
-- 
2.25.1

